### ISSUE TITLE
What's the pre-processing of DIV2K for training? missing contents in crop_***.py

### ISSUE THREAD

**Issue by Zirconium2159** [Yang-Liu1082/InvDN#3]
I tried to perform training on DIV2K dataset with the help of LQGTSN_dataset.py. However, the performance seems poor(under 30dB PSNR, quite lower than proposed). 
The loss curves also differ a lot from the training on SIDD dataset. Maybe I missed some important configurations when training on the DIV2K. 
It will be appreciated if you provide corresponding pre-trained models or more detailed instructions for reproduction.
It seems empty in crop_SIDD.py. Besides, crop_DIV2K.py doesn't match the DIV2K dataset.

**Comment by Yang-Liu1082** [comment]
Thank you for your interest in our work. The results reported in the paper are trained on the SIDD dataset with 'LQGTRN_dataset.py'. 'crop_SIDD.py' is only a function to crop the SIDD images into smaller patches. No special preprocessing has been done for that. This file as well as the pre-trained models will be updated soon.
